openstf/model/objective.py

```diff
@@ -31,19 +33,83 @@ class RegressorObjective:
         study.optimize(objective)
     """
 
+    def process_tuple(
+        self,
+        trial: optuna.trial.FrozenTrial,
+        key: str,
+        value: Tuple[Union[int, float], Union[int, float]],
+    ) -> Union[int, float]:
+        """Pick hyperparameter for tuple"""
```

Comment:
I will probably have to read the code a bit further. But it is not immediately clear what this function does? It does some processing on a tuple and/or it picks a hyperparameter for a tuple. Probably a good idea to bring the method name more in line with the description now it looks like two different things.